unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Quest_V2/Tests/QuestOfferComponentViewTests.cs

```diff
@@ -0,0 +1,89 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace DCL.Quest
+{
+    public class QuestOfferComponentViewTests
```

Comment:
Nitpicking but tests should follow our pattern "QuestOfferComponentViewShould" to have meaningfull names.